libbpf-tools/gethostlatency.bpf.c

```diff
@@ -60,13 +60,13 @@ static int probe_return(struct pt_regs *ctx)
 	return 0;
 }
 
-SEC("kprobe/handle_entry")
+SEC("uprobe")
 int BPF_KPROBE(handle_entry)
```

Comment:
We can also replace `BPF_KPROBE` with `BPF_UPROBE`.

Reply:
I thought about this as well. I didn't make the change because I don't see any other usage of `BPF_UPROBE` in this codebase. 

FYI, the macros `BPF_UPROBE` and `BPF_URETPROBE` are added in [libbpf v1.2.0](https://github.com/libbpf/libbpf/releases/tag/v1.2.0) released on May 1, 2023. That's not an issue if people build libbpf-tools using the libbpf submodule under `src/cc/libbpf`.